src/test/java/com/tom/driver/manager/DriverManager.java

```diff
@@ -2,26 +2,43 @@
 
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.util.HashMap;
+import java.util.Map;
 
 public class DriverManager {
     private static WebDriver driver;
-
+    public static String DOWNLOAD_FILE_FOLDER = "C:\\Users\\Tomek\\IdeaProjects\\tmp";
     private DriverManager() {
     }
 
     public static WebDriver getWebDriver() {
 
         if (driver == null) {
             System.setProperty("webdriver.chrome.driver", "C:/drivers/chromedriver.exe");
-            driver = new ChromeDriver();
+            driver = new ChromeDriver(setupDownloadFolder(DOWNLOAD_FILE_FOLDER));
         }
 
         return driver;
     }
 
-    public static void disposeDriver(){
+    public static void disposeDriver() {
         driver.close();
         driver.quit();
         driver = null;
     }
+
+    public static ChromeOptions setupDownloadFolder(String path) {
+        Map<String, Object> prefsMap = new HashMap<String, Object>();
```

Comment:
java 11 ma prostszy sposób tworzenia takich małym map: Map.of(key1.value1,key2,value2...)

Reply:
Używam javy 8